src/utils/unit_test/random_fill_test.cpp

```diff
@@ -181,6 +181,9 @@ struct TensorDataTypeStruct<El::DistMatrix<T, ColDist, RowDist, Wrap, D>>
 template <typename DistMat>
 using TensorDataType = typename TensorDataTypeStruct<DistMat>::type;
 
+// Test disabled on 08/16/23 due to a consistent failure that cannot be
+// reproduced outside CI (and a single system at that)
+#if 0
```

Comment:
@tbennun Please add an issue in the issue tracker so that we can come back later and figure out if we need to reassess this test later.

Reply:
#2307